README.md

```diff
@@ -14,3 +15,57 @@ In addition, we are trying to have a well-designed structure ourselves so that v
 
 Through these activities, ONE-vscode can provide a differentiation that cannot be experienced only with the existing CLI by linking the toolchain with the execution environment, ONE Runtime, or a target simulator corresponding to the backend compiler. This will eventually lead to fun and high productivity for developers.
 
+## Getting started
+
+- [How to install?](./docs/HowToInstall.md)
+- [How to use?](./docs/HowToUse.md)
+- [Terms](./docs/Terms.md)
+
+## Extension features
+
+You can see `ONE` logo on Acitivity Bar. (If you are not familiar with terms of vscode, you can read [document](https://code.visualstudio.com/docs/getstarted/userinterface).)
+
+<img src="https://user-images.githubusercontent.com/10216715/174931217-2eaf6091-4b0d-4d9d-a9d0-f44b80300e71.png" width=500 />
+
+###  One view explorer
+
+ONE-vscode supports an specialized explorer for NN model files and config files.
+
+<img src="https://user-images.githubusercontent.com/10216715/174931232-62ce9853-c153-4c27-b94d-1a885eb4bef2.png" width=500 />
+
+###  Toolchain view explorer
+
+ONE-vscode supports an specialized explorer for ONE toolchain.
+
+<img src="https://user-images.githubusercontent.com/10216715/174931237-bdedf079-bd2d-4600-8697-f171f87ea159.png" width=500 />
+
+### Compile
+
+ONE-vscode supports compiling NN model by a config file with ONE toolchain.
+
+<img src="https://user-images.githubusercontent.com/10216715/174931224-4039c32a-aea7-4a1a-bd86-cb30edb8831e.png" width=500 />
+
+### Config editor
+
+ONE-vscode supports an config editor for config files.
+
+<img src="https://user-images.githubusercontent.com/10216715/174931229-c0649c60-0308-4916-8ee7-691c30a7859b.png" width=500 />
+
+### Target devices view explorer
+
+(NYI)
+
+<img src="https://user-images.githubusercontent.com/10216715/174931234-73dd7513-f8c7-49f7-8675-9cec05b247f2.png" width=500 />
+
+### In the future
+
+- Value test
+- Profile
+
```

Comment:
```suggestion
## Overview and tutorials

- [ONE-vscode extension overview](./docs/Overview.md)
- [How to install?](./docs/HowToInstall.md)
- [How to use?](./docs/HowToUse.md)
- [Terms](./docs/Terms.md)

## Features

<img src="https://user-images.githubusercontent.com/7223627/175224054-0df3109a-f4ef-4a12-a7be-ab50f01b77af.png" width=500 />

- Install toolchain

<img src="https://user-images.githubusercontent.com/7223627/174947559-71ab213c-68c4-4700-bcec-e55f6a2203bf.gif" width=500 />

- Compile NN model using config file

- Setup available NPU backend device

- Inference backend model

- Profile backend model

```

Reply:
And add `Overview` page in docs.

`Overview.md`
```
# **ONE**-vscode tutorial

In this tutorial, you will use `ONE` driver to compile a model, and refer to and profile the generated backend model in Visual Studio Code. By using the ONE-vscode extension, you make VS Code into a great ONE IDE.

`ONE` driver is a command tool that generates binary running in the NPU backend, executes it and profiles it. Therefore, NPU backend is required to use ONE driver. The currently available backends are as follows:
- TRIX
If you want to use your NPU backend in ONE vscode, this can be implemented by redefining the Backend very simply. See the following guide, [How to define new Backend for ONE-vscode](docs/HowToDefineNewBackend.md), for help.

This tutorial introduces you to VS Code as a ONE environment, primarily how to compile a model, inference and profile a generated model and show the profile data through the following tasks:
- Install toolchain
- Compile NN model using config file
- Setup available NPU backend device
- Inference backend model
- Profile backend model

## UX Guidelines

<img src="https://user-images.githubusercontent.com/7223627/175224054-0df3109a-f4ef-4a12-a7be-ab50f01b77af.png" width=500 />

If you are not familiar with terms of vscode, you can read [userinterface](https://code.visualstudio.com/docs/getstarted/userinterface) in vscode.

### One view container

### One explorer view

### Toolchain explorer view

### Device explorer view

### Output log view

## Features

### Install toolchain

<img src="https://user-images.githubusercontent.com/7223627/174947559-71ab213c-68c4-4700-bcec-e55f6a2203bf.gif" width=500 />

### Compile NN model using config file

### Setup available NPU backend device

### Inference backend model

### Profile backend model
```